<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>159</width>
     <height>42</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Concede</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playerLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>157</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="rulesGraphics">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>20</y>
     <width>80</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hint1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotates the arrow 45 degrees clockwise at the end of each turn</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>67</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotates the arrow 45 degrees counterclockwise at the end of each turn</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>123</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lets the arrow move two squares each turn</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Upgrades the arrow to home on enemies</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="boardGraphics">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>159</width>
     <height>42</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Toggle Hints</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
